content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -289,6 +289,6 @@ For more information on the fields and usage of the configuration you can naviga
 
 ## {{% heading "whatsnext" %}}
 
-* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init/) to bootstrap a Kubernetes control-plane node
-* [kubeadm token](/docs/reference/setup-tools/kubeadm/kubeadm-token/) to manage tokens for `kubeadm join`
-* [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset/) to revert any changes made to this host by `kubeadm init` or `kubeadm join`
+* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init/) to bootstrap a Kubernetes control-plane node.
```

Comment:
do we have  "." for all items on all pages? if not i don't we should be changing this.

Reply:
we have "." for all items on thisl page.